lips/lip-1.mdx

```diff
@@ -183,7 +123,7 @@ All requests between VASPs are structured as `CommandRequestObject`s.
 | _ObjectType  | str    | Y | Fixed value: `CommandRequestObject`|
 | command_type | str    | Y | A string representing the type of Command contained in the request. |
 | command      | Command object | Y | The Command to sequence. |
-| cid | str    | Y     | A unique identifier for the Command. |
+| cid | str    | Y     | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122). |
```

Comment:
I can't recall why we need this anymore.  Isn't the header covering this now?

Reply:
I think header x-request-id is per request, but the cid can be same if the command was processed with error and client side retried the command with a new http request

Reply:
Discussed offline and cid is largely for debugging purposes right now.  It allows replayed requests with different request IDs to be seen as a single command and traced against eachtoehr